sql-plugin/src/main/340/scala/org/apache/spark/sql/execution/datasources/parquet/rapids/shims/ParquetCVShims.scala

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
really this hsould probably be in a 340+ directory but it seems we have other things in just 340 so I guess we can fix it when we add in 3.4.1 or 3.5.0

Reply:
Yes, that's my plan.